layouts/partials/menu.html

```diff
@@ -7,25 +7,51 @@
           {{ range .Site.Menus.main }}
               {{ if .HasChildren }}
 
-            <li class="sub-menu{{if $currentNode.HasMenuCurrent "main" . }} active{{end}}">
-            <a href="javascript:;" class="">
-                {{ .Pre }}
-                <!--<i class="icon_desktop"></i>-->
-                <span>{{ .Name }}</span>
-                <span class="menu-arrow fa {{if $currentNode.HasMenuCurrent "main" . }}fa-angle-down{{else}}fa-angle-right{{end}}"></span>
-            </a>
-                <ul class="sub{{if $currentNode.HasMenuCurrent "main" . }} open{{end}}">
-                    {{ range .Children }}
-                    <li{{if $currentNode.IsMenuCurrent "main" . }} class="active"{{end}}><a href="{{.URL}}">{{ .Name }}</a> </li>
-                    {{ end }}
-                </ul>
-              {{end}}
+              <li class="sub-menu{{if $currentNode.HasMenuCurrent "main" . }} active{{end}}">
+                  {{ if .HasChildren }}
+                  <a href="javascript:;" class="">
+                      {{ .Pre }}
```

Comment:
The indentation looks off on this line, I think it needs to be pulled one space to the left.